go/internal/multipeer-connectivity-transport/driver/bridge.go

```diff
@@ -0,0 +1,34 @@
+// +build darwin
+
+package driver
+
+import (
+	native "berty.tech/berty/v2/go/internal/multipeer-connectivity-transport/driver/mc-driver"
+)
+
+// Native -> Go functions
+func BindNativeToGoFunctions(hfp func(string) bool, rfp func(string, []byte)) {
+	native.GoHandleFoundPeer = hfp
+	native.GoReceiveFromPeer = rfp
+}
+
+// Go -> Native functions
+func StartMCDriver(localPID string) bool {
```

Comment:
You shouldn't return a bool here if the return of this func is ignored by the transport

Reply:
done!